src/components/authentication/Login.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import { Link } from 'react-router-dom';
+import "bootstrap/dist/css/bootstrap.css";
+import Logo from "../../images/logo.png";
+
+const Login = () => {
+  // State for storing username and password
+  const [username, setUsername] = useState({
+    value: '',
+    error: false,
+  });
+  const [password, setPassword] = useState({
+    value: '',
+    error: false,
+  });
+
+  const handleBlur = (value, setState) => {
+    if (value.trim() === '') {
+      setState({ value, error: true });
+    }
+    else {
+      setState({ value, error: false });
+    }
+  };
+
+  // Handle the form submission
+  const handleSubmit = (event) => {
+    console.log({ username, password });
```

Comment:
please remove all the console log statements. I asked you to remove these before too.